deploy/docker/pump.io.json

```diff
@@ -0,0 +1,23 @@
+{
+    "driver":  "mongodb",
+    "params":  {"dbname": "pumpio", "host":"127.0.0.1", "port":27017},
+    "secret":  "put a secret string here",
+    "noweb":  false,
+    "site":  "My pump.io site using Docker",
+    "owner":  "My name is Docker",
+    "ownerURL":  "http://dockerio.cloudapp.net",
+    "hostname":  "pumpio.example.com",
+    "port":  443,
+    "urlPort": 443,
```

Comment:
Unless you're running nginx inside the Docker container, it won't matter. The reason `urlPort` exists is so that you can specify `443` in URLs but still bind to some high-numbered port. The only reason you'd need to bind to a different port is because something (i.e. nginx) is already using port 443 on the system's primary network interface.

This isn't relevant in the Docker case since the container will get its own private network with its own private port 443. So `urlPort` doesn't matter no matter what.